test/wasm_worker/thread_stack.c

```diff
@@ -1,43 +1,42 @@
-#include <emscripten.h>
+#include <emscripten/console.h>
 #include <emscripten/wasm_worker.h>
 #include <emscripten/stack.h>
 #include <stdio.h>
 #include <assert.h>
 #include <stdlib.h>
 
-#define THREAD_STACK_SIZE 1024
+#define THREAD_STACK_SIZE 2048
```

Comment:
Stack size change because of `emscripten_outf`?

Reply:
Yup, essentially printf.  The exact side of the stack in this test is not super important though I think.  I just designed to ensure that stack start/end match our expectations.